crates/router/tests/connectors/sample_auth.toml

```diff
@@ -183,4 +183,9 @@ api_key="API Key"
 api_key="API Key"
 
 [prophetpay]
-api_key="API Key"
\ No newline at end of file
+api_key="API Key"
+
+[bankofamerica]
+api_key = "Bearer MyApiKey"
```

Comment:
Can you please remove this bearer, we will be concatenating `bearer` in connector.rs?

Reply:
Done